docs/cli_usage_guide.md

```diff
@@ -50,19 +50,23 @@ Many of the arguments for this command have set defaults or will use an environm
 Usage: firewatch report [OPTIONS]
 
 Options:
+  --keep-job-dir                If set, firewatch will not delete the job
```

Comment:
Might want to consider:
`--keep-job-dir BOOL`

`--fail_with_test_failures BOOL`

Reply:
I have updated, this is just the way click formats their help output.